core/controllers/access_validators.py

```
@@ -875,3 +875,33 @@ class ReviewTestsPageAccessValidationHandler(
+    def get(self) -> None:
+        """Handles GET requests."""
+        pass
+
+
+class PlatformFeedbackReportsAccessValidationHandler(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Validates access to Platform Feedback Reports."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)